benchexec/benchexec.py

```diff
@@ -136,6 +136,13 @@ def create_argument_parser(self):
                                'use "-1" to disable time limits completely)',
                           metavar="SECONDS")
 
+        parser.add_argument("-W", "--walltimelimit",
+                            dest="walltimelimit", default=None,
+                            help='Wall time limit for each run, e.g. "90s" '
+                                 '(overwrites wall time limit from XML file, '
+                                 'use "-1" to disable time limits completely)',
```

Comment:
This is not correct, right? If -1 is passed, the wall-time limit is `None`, which means that the standard wall-time limit will be used. Or did I overlook something?

(In any case, the text that says 'use "-1" to disable time limits completely' is no longer correct, of course.)

Reply:
Yes, sorry - this is a mistake.

What would you like to see as a default behavior? Disabling the limit completely or using the default value which is used currently? And seems that currently all timelimit options do not accept -1. Is this a bug or an intended scenario and help is incorrect?

Reply:
Hm, -1 should work actually, and it does for me:
```
benchexec doc/benchmark-example-rand.xml --timelimit -1
```
What exactly does not work?

For wall-time limit, I would say the behavior of -1 should be the current default value.

Reply:
I see, seems that I have introduced a bug to this somehow too. Ok, I will fix -1 and wall option then.